server/routes/automationRouter.js

```diff
@@ -5,6 +5,6 @@ import automationStatusController from '../controllers/automationStatsController
 const router = express.Router();
 
 router.get('/', automationController.getAutomations);
-router.get('/stats', automationStatusController);
+router.get('/stats/:id', automationStatusController);
```

Comment:
Nice job. Why was the parameter named `id`? Does it represent an `id`?

Reply:
I think it would be better to let this endpoint remain as: `/stats`. The `days, weeks, months and year` parameters can be passed a `span` argument, just as we have `date` argument

Reply:
Thanks @nzediegwu1 , I have added the `duration` as a query in the URL instead of a parameter.